lite/examples/style_transfer/ios/StyleTransfer/ModelDataHandler/CombinedModelDataHandler.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  CombinedModelDataHandler.swift
+//  StyleTransfer
+//
+//  Created by Ivan Cheung on 3/28/20.
+//  Copyright © 2020 Ivan Cheung. All rights reserved.
+//
+
+import TensorFlowLite
+
+/// This class handles all data preprocessing and makes calls to run inference on a given frame
+/// by invoking the `Interpreter`. It then formats the inferences obtained and returns the top N
+/// results for a successful inference.
+class CombinedModelDataHandler: ModelDataHandling {
+  // MARK: - Internal Properties
+  var inputWidth: Int { styleTransferModelDataHandler.contentImageSize }
+  var inputHeight: Int { styleTransferModelDataHandler.contentImageSize }
+  let threadCount: Int
+  let threadCountLimit = 10
+  
+  // MARK: - Private Properties
+  let stylePredictorModelDataHandler: StylePredictorModelDataHandler
+  let styleTransferModelDataHandler: StyleTransferModelDataHandler
+  
+  // TODO: Cache last style predictor to use if not changed
+    
+  // MARK: - Initialization
+
+    
+  /// A failable initializer for `ModelDataHandler`. A new instance is created if the model and
+  /// labels files are successfully loaded from the app's main bundle. Default `threadCount` is 1.
+  init?(threadCount: Int = 1) {
```

Comment:
It'd be nice to figure out best practices for chaining model inferences together. This is my approach to composing the prediction and transfer inference steps.